### ISSUE TITLE
Small README fix - no import of MBAR 

### ISSUE THREAD

**Issue by atharva-kelkar** [choderalab/pymbar#434]
Hi,

In the README, there should be the following line:

`>>> from pymbar import testsystems, MBAR`

instead of only

`>>> from pymbar import testsystems`

The MBAR class used in the trial problem but not imported

**Comment by mikemhenry** [comment]
@atharva-kelkar Thank you for brining this to our attention! Are you interested in making a pull request (PR) to fix this error?

**Comment by atharva-kelkar** [comment]
Yes, I will do that. Some other typos/outdated things in the README as well - I will also fix those.

**Comment by mikemhenry** [comment]
Thanks!
